OsmAnd/src/net/osmand/plus/nearbyplaces/NearbyPlacesFragment.java

```diff
@@ -0,0 +1,87 @@
+package net.osmand.plus.nearbyplaces;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.ColorRes;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.FragmentActivity;
+import androidx.fragment.app.FragmentManager;
+
+import net.osmand.PlatformUtil;
+import net.osmand.plus.R;
+import net.osmand.plus.activities.MapActivity;
+import net.osmand.plus.base.BaseOsmAndFragment;
+import net.osmand.plus.helpers.AndroidUiHelper;
+import net.osmand.plus.search.dialogs.QuickSearchDialogFragment;
+import net.osmand.plus.utils.AndroidUtils;
+import net.osmand.plus.utils.ColorUtilities;
+
+import org.apache.commons.logging.Log;
+
+public class NearbyPlacesFragment extends BaseOsmAndFragment {
```

Comment:
We need this only for toolbar? better create something similar to QuickSearchToolbarController